README.md

```diff
@@ -136,10 +136,10 @@ DREX also comes with a simple project drawer functionality
 
 ## Configuration
 
-There is no initial setup needed to use DREX  
```

Comment:
These trailing white spaces are there on purpose to force a line break in markdown